app/constants/list.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const VISIBILITY_CONFIG_DEFAULTS = {
+    itemVisiblePercentThreshold: 100,
```

Comment:
I'm guessing that to count a category as viewable, the whole text should be clearly visible.